### ISSUE TITLE
Aria-required attribute is not working for picker

### ISSUE THREAD

**Issue by ravi-405** [microsoft/fluentui#14937]
User Experience:
Users who depends on screen reader are facing difficulty as input field is mandatory for "People picker".

Repro Steps:
Step 1: Open https://fabricweb.z5.web.core.windows.net/oufr/7.105.2/#/examples/peoplepicker
Step 2:  Input element(TextField) is not having aria-required attribute even though we send it as prop.

Actual Result:
Input element(TextField) is not having aria-required attribute even though we send it as prop.

Expected Result:
Input element(TextField) should have aria-required attribute if we send it as prop.

**Comment by miroslavstastny** [comment]
@ravi-405 Could you please add a codepen example showing what does not work for you?

In the link you have shared, if I edit the example to add `aria-required` to `inputProps`:

```jsx
<NormalPeoplePicker
  ...
  inputProps={{
    'aria-required': true,          
  }}
/>
```
The `aria-required` is propagated to the `input` element and is correctly recognized in Accessibility Object Model:

![image](https://user-images.githubusercontent.com/9615899/92494964-9e66af80-f1f6-11ea-9924-f19185036034.png)

**Comment by ravi-405** [comment]
@miroslavstastny , Thanks for the help. It is working fine as expected now.
